### PR TITLE
add AVKit and MultipeerConnectivity framework

### LINKED ISSUE #706: Library not loaded: /System/Library/Frameworks/AVKit.framework/Versions/A/AVKit
When I tried to run an application.
I got an error like this:
`
dyld: Library not loaded: /System/Library/Frameworks/AVKit.framework/Versions/A/AVKit
  Referenced from: /Applications/xxx.app/Contents/MacOS/xxx
  Reason: image not found
Abort trap: 6 (core dumped)
`

### PR DESCRIPTION
**Pull request by Yingsheng-eroad**
fix #706 add AVKit and MultipeerConnectivity framework

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -379,6 +380,7 @@ add_subdirectory(private-frameworks)
 add_subdirectory(external/usertemplate)
 add_subdirectory(AVKit)
 add_subdirectory(MultipeerConnectivity)
+add_subdirectory(CoreGraphics)
```
Comment (ahyattdev) [inline src/CMakeLists.txt]: ~~We have CoreGraphics in src/external/cocotron/CoreGraphics~~ seems like this is already taken care of

**Comment by CuriousTommy** [comment]
@ahyattdev I'll update the wiki to make note of the `src/frameworks` directory

**Comment by Yingsheng-eroad** [comment]
> These stubs look good. My only ask is that they be moved into `src/frameworks`.
> 
Thanks, resolved.

File: src/frameworks/CMakeLists.txt [hunk 2]
```diff
@@ -20,3 +20,5 @@ add_subdirectory(CoreServices)
 add_subdirectory(Kerberos)
 add_subdirectory(MapKit)
 add_subdirectory(OpenAL)
+add_subdirectory(AVKit)
```
Comment (CuriousTommy) [inline src/frameworks/CMakeLists.txt]: Could you put this in alphabetical order.

**Pull request merged**